app/src/main/java/swati4star/createpdf/fragment/ExceltoPdfFragment.java

```diff
@@ -192,7 +192,7 @@ public void openExcelToPdf() {
 
     @OnClick(R.id.open_pdf)
     void openPdf() {
-        mFileUtils.openFile(mPath);
+        mFileUtils.openFile(mPath, FileUtils.FileType.e_PDF);
```

Comment:
why these changes?

Reply:
For listing files and directories filters.

Reply:
This PR is supposed to add marathi locale only, right? Then, these changes should not be a part of this PR.